### ISSUE TITLE
Walrus Atmosphere Curve

### ISSUE THREAD

**Issue by hemeac** [post-kerbin-mining-corporation/NearFutureLaunchVehicles#109]
The atmosphere curve of the walrus (nflv-engine-ar1-1) doesn't approach 0. Will update this issue if I come across others in LV lineup.

```
atmosphereCurve
		{
			key = 0 315
			key = 1 285
			key = 12 100
		}
```

**Comment by hemeac** [comment]
The Manatee has the same atmosphere curve as well nflv-engine-ar1c-1

**Comment by ChrisAdderley** [comment]
Fixed for 2.1.3
